src/rules/__tests__/require-hook.test.ts

```diff
@@ -374,6 +386,25 @@ ruleTester.run('require-hook', rule, {
         },
       ],
     },
+    {
+      code: dedent`
+        enableAutoDestroy(afterEach);
+        
+        describe('some tests', () => {
+          it('is false', () => {
+            expect(true).toBe(true);
+          });
+        });
+    `,
+      options: [{ allowedFunctionCalls: ['someOtherName'] }],
```

Comment:
```suggestion
        });
      `,
      options: [{ allowedFunctionCalls: ['someOtherName'] }],
```

Reply:
Got it. Is there any automatic linting for that? Neither prettier nor eslint fixed this.

Reply:
Sadly not because technically it affects the content (since you're adding more spaces inside the string) but we strip it away using `dedent`, which eslint & prettier don't understand.